remove logspam
ERROR: no OSD provider available - using dummy OSD!

--- a/osd.c
+++ b/osd.c
@@ -2302,8 +2302,6 @@ cOsd *cOsdProvider::NewOsd(int Left, int Top, uint Level)
         }
      return Osd;
      }
-  else
-     isyslog("no OSD provider available - using dummy OSD!");
   return new cOsd(Left, Top, 999); // create a dummy cOsd, so that access won't result in a segfault
 }
 
--- a/vdr.c
+++ b/vdr.c
@@ -777,11 +777,6 @@ int main(int argc, char *argv[])
   CamResponsesLoad(AddDirectory(ConfigDirectory, "camresponses.conf"), true);
   DoneRecordingsPattern.Load(AddDirectory(CacheDirectory, "donerecs.data"));
 
-  if (!*cFont::GetFontFileName(Setup.FontOsd)) {
-     const char *msg = "no fonts available - OSD will not show any text!";
-     fprintf(stderr, "vdr: %s\n", msg);
-     esyslog("ERROR: %s", msg);
-     }
 
   // Recordings:
